distribution/sender/routes/sync-response-processor.xml

```diff
@@ -0,0 +1,66 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="sync-response-processor" errorHandlerRef="shutdownErrorHandler">
+
+        <from uri="scheduler:sync_response_processor?initialDelay={{sync-response-processor.initial.delay}}&amp;delay={{sync-response-processor.delay}}" />
+
+        <log message="Fetching next {{sync-response-processor.max.fetch.results}} sync responses" loggingLevel="DEBUG" />
```

Comment:
Please use 1000 and remove this sync-response-processor.max.fetch.results property

Reply:
Changed to 500, as mentioned in [this comment](https://github.com/FriendsInGlobalHealth/openmrs-eip/pull/11#discussion_r869835394), and property was removed.